UPGRADE-4.2.md

```diff
@@ -83,6 +83,11 @@ FrameworkBundle
    this will generate 404s for non-UTF-8 URLs, which are incompatible with you app anyway,
    and will allow dumping optimized routers and using Unicode classes in requirements.
 
+Messenger
+---------
+
+ * The `Symfony\Component\Messenger\Middleware\ValidationMiddleware` and `Symfony\Component\Messenger\Asynchronous\Middleware\SendMessageMiddleware` do not support messages in their `handle` method anymore because they implement the `EnvelopeAwareInterface`. You need to give them an instance of the `Envelope` object, wrapping your message. You can use Envelope::wrap($message).
```

Comment:
Add ` around the `Envelope::wrap` example. 

Also, I think that I'd phrase it like: 
> The `handle` method of the X and Y middleware now requires an `Envelope` object to be given (because they implement the `EnvelopeAwareInterface`. When using these middleware with the provided MessageBus, you will not have to do anything. If you use the middleware any other way, you can use `Envelope::wrap($message)` to create an envelope for your message.

Reply:
It looks a lot better like that.